modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/components/Body/components/GoalsEntries/GoalsEntries.tsx

```diff
@@ -69,13 +72,11 @@ const GoalsEntries = ({mdfRequest}: IProps) => (
 				},
 				{
 					title: 'Liferay business/sales goals',
-					value:
-						mdfRequest.liferayBusinessSalesGoals !== undefined &&
-						mdfRequest.liferayBusinessSalesGoals.includes(
-							'Other - Please describe'
-						)
-							? mdfRequest.liferayBusinessSalesGoalsOther
-							: mdfRequest.liferayBusinessSalesGoals?.join('; '),
+					value: mdfRequest.liferayBusinessSalesGoals?.join('; '),
+				},
+				{
+					title: 'Liferay business/sales goals Other',
+					value: mdfRequest.liferayBusinessSalesGoalsOther,
```

Comment:
It can be a kind confusing have another section just for the other option. The best it's to use the same section.

`value: mdfRequest.liferayBusinessSalesGoalsOther || mdfRequest.liferayBusinessSalesGoals?.join('; ')`